solo/client.py

```diff
@@ -28,6 +28,7 @@
 import solo.exceptions
 from solo import helpers
 from solo.commands import SoloBootloader, SoloExtension
+import solo.fido2
```

Comment:
Is this needed here?

Reply:
Yes. If I comment out that import and change nothing else on my machine I get the following error (after installing that version from source):

```
$ python
Python 3.8.3 (default, May 19 2020, 23:51:58) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import solo
>>> key = solo.client.find()
>>> sn = key.dev.descriptor['serial_number']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'serial_number'
```

If that import is running the serial number is read properly. I am not sure how I ended up finding that I needed that import, it's been some time but I remember that it was a strange discovery. By looking at the code [this line](https://github.com/Iolaum/solo-python/blob/linux_parseuevent/solo/client.py#L37) stands out but it's not one of the things this patch changes.